src/views/Invite/InviteEmail.js

```diff
@@ -45,13 +51,90 @@ const EMAIL_STATUS = {
   FAIL: '×',
 };
 
+// world's simplest uid
+let id = 0;
+
+const buildInputConfig = (extra = {}) =>
+  buildEmailInputConfig({ name: (id++).toString(), ...extra });
```

Comment:
Probably want `'email' + (id++).toString()`?

Reply:
@Fang- not necessary, since ids only need to be valid within their own loop, unless I misinterpreted your reason

Reply:
My reasons are purely semantic. This is what ends up in the element's `name` attribute, right? It seems cleaner to have something meaningful in there, rather than just a number. But you're right that it won't, technically, matter.

Reply:
ah, totally - will add it